source/win32_tinyengine.c

```diff
@@ -188,46 +1242,90 @@ WinMain(HINSTANCE Instance, HINSTANCE PrevInstance, LPSTR CommandLine, int ShowC
         HWND Window = CreateWindowExW(0, WindowClass.lpszClassName, L"TinyEngine", WS_OVERLAPPEDWINDOW, CW_USEDEFAULT, CW_USEDEFAULT, CW_USEDEFAULT, CW_USEDEFAULT, 0, 0, Instance, 0);  // NOTE(zak): At somepoint the window name should reflect the applications name
         if(Window != INVALID_HANDLE_VALUE)
         {
-            if(Win32InitD3D11(Window))
+            // Input
+            RAWINPUTDEVICE RawInputDevices[NUM_RAW_INPUT_DEVICES];
+            { 
+                // Mouse
+                RawInputDevices[0].usUsagePage = 0x01;
+                RawInputDevices[0].usUsage = 0x02;
```

Comment:
Maybe use the flag to disable legacy messages for the keyboard. Can't think of a reason we'd need them